api/leaderboard_test.go

```diff
@@ -2535,6 +2535,6 @@ var _ = Describe("Leaderboard Handler", func() {
 			status, body, err := fastGet(url)
 			Expect(err).NotTo(HaveOccurred())
 			Expect(status).To(Equal(http.StatusOK), string(body))
-		}, 0.1)
+		}, 0.9)
```

Comment:
How the new leaderboard uses GO to execute logic instead of redis, the time to execute the new flow is greater than with lua script on redis. But how the SLA is around 80ms and currently we have response time in 8ms. I think that is ok resize up this performance test.